### PR TITLE
New package: fmit-1.1.13

### PR DESCRIPTION
**Pull request by cr6git**
Free Music Instrument Tuner 🎼

### CODE REVIEW

**Comment by cr6git** [comment]
Oopss... According to [INSTALL.txt](https://github.com/gillesdegottex/fmit/blob/master/INSTALL.txt#L26) I forgot to add the qmake options. 
Sorry, I will change that right now.

**Comment by cr6git** [comment]
it's done.

File: srcpkgs/fmit/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+do_configure() {
+	qmake "CONFIG+=acs_qt acs_alsa acs_jack acs_portaudio" \
+		PREFIX=/usr \
+		LIB=/usr/lib \
+		QMAKE_CC=$CC QMAKE_CXX=$CXX QMAKE_LINK=$CXX \
+		QMAKE_CFLAGS="${CFLAGS}" \
+		QMAKE_CXXFLAGS="${CXXFLAGS}" \
+		QMAKE_LFLAGS="${LDFLAGS}" \
+		fmit.pro
+}
```
Comment (Vaelatern) [inline srcpkgs/fmit/template]: Do you need to add `fmit.pro` to the end? Or can you fit the `"CONFIG+` string and `fmit.pro` into `configure_args` and use the build style?

Comment (cr6git) [reply srcpkgs/fmit/template]: I tried using `configure_args="CONFIG+=acs_qt acs_alsa acs_jack acs_portaudio"` but it doesn't work: 
````
=> fmit-1.1.13_1: running do_configure ...
Cannot find file: acs_alsa.
Cannot find file: acs_jack.
Cannot find file: acs_portaudio.
=> ERROR: fmit-1.1.13_1: failed to run do_configure() at line 17.
````
seems like `xbps-src` is not managing the quotes properly.

**Comment by newbluemoon** [comment]
What does work is

    configure_args="CONFIG+=acs_qt CONFIG+=acs_alsa CONFIG+=acs_jack CONFIG+=acs_portaudio"

But it looks a bit strange.

**Comment by cr6git** [comment]
@newbluemoon thanks for the information! 

@Vaelatern do you agree?

**Comment by newbluemoon** [comment]
Found the right sequence to escape the quotation marks:

    configure_args="\\\\"CONFIG+=acs_qt acs_alsa acs_jack acs_portaudio\\\\""

Does this look better? ;-)

**Comment by cr6git** [comment]
Personally, I prefer your previous proposal...

**Pull request merged**